v22.1/backup-and-restore-overview.md

```diff
@@ -0,0 +1,61 @@
+| [Locality-aware backup and restore](take-and-restore-locality-aware-backups.html) | A backup where each node writes files only to the backup destination that matches the node locality configured at node startup.                              | {{ site.data.products.dedicated }} or with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) in CockroachDB.                                                                                                                                       |
+| [Scheduled backup](manage-a-backup-schedule.html)                  | A schedule for periodic backups.                                                                                                                             | {{ site.data.products.dedicated }} or with an [{{ site.data.products.enterprise }} license](enterprise-licensing.html) in CockroachDB  
+| [Automated Backups](../cockroachcloud/backups-page.html)                 | Cockroach Labs runs full backups daily and incremental backups hourly for every CockroachDB Cloud cluster.                                                   | {{ site.data.products.db }} clusters                                                                                                                                     
+
+## Backup and restore SQL statements
+
+The following table outlines SQL statements you can use to create, configure, pause, and show backup and restore jobs:
+
+| SQL Statement  | Description                                                                                 |
+|----------------|---------------------------------------------------------------------------------------------|
+| [`ALTER BACKUP`](alter-backup.html) | Add a new [KMS encryption key](take-and-restore-encrypted-backups.html#use-key-management-service) to an encrypted backup.                                        |
+| [`BACKUP`](backup.html)       | Create full and incremental backups.                                                        |
+| [`CANCEL JOB`](cancel-job.html)   | Use with the `job ID` to cancel a backup or restore job.                                    |
+| [`PAUSE JOB`](pause-job.html)    | Use with the `job ID` to pause a backup or restore job.                                     |
+| [`RESTORE`](restore.html)      | Restore full and incremental backups.                                                       |
+| [`RESUME JOB`](resume-job.html)   | Use with the `job ID` to resume a backup or restore job.                                    |
+| [`SHOW BACKUP`](show-backup.html)  | Show a backup's details at the [backup collection's](take-full-and-incremental-backups.html#backup-collections) storage location.                        |
+| [`SHOW JOBS`](show-jobs.html)    | Use to show a list of jobs or with the `job ID` to show details of a backup or restore job. |
+
```

Comment:
These are alphabetically ordered here, but would it make sense to order them in a the order of usage patterns?
e.g. 

BACKUP - start a backup
SHOW JOBS - show jobs to see status of my backup job
PAUSE/RESUME/CANCEL JOB -- pause/resume' cancel my backup job
SHOW BACKUP -- show details of a complete backup
RESTORE -- restore from a completed backup
ALTER BACKUP -- if kms rotated keys, add new keys to existing chain